gems/smithy/lib/smithy/polishes/rubocop.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Polishes
+    # Runs RuboCop autocorrect-all on the generated projections.
+    class RuboCop < Polish
+      def polish(_artifact)
```

Comment:
A nit - this is an enumerable right? Should this be `artifacts`? (obviously doesn't matter here since unused, just a general question on polishes).